docs/modules/ROOT/pages/how-to/remove-default-networkpolicies.adoc

```diff
@@ -16,25 +21,21 @@ You can disable the automatic network policy management of {product} by adding l
 apiVersion: v1
 kind: Namespace
 metadata:
-  annotations:
-    openshift.io/description: ""
-    openshift.io/display-name: My cool project
-    openshift.io/requester: my-username
   labels:
     appuio.io/organization: my-company
     kubernetes.io/metadata.name: my-namespace
     network-policies.syn.tools/no-defaults: 'true' <1>
-    network-policies.syn.tools/purge-defaults: 'true' <2>
+    set.network-policies.syn.tools/base: '' <2>
   name: my-namespace
 ----
-<1> Add this label to prevent {product} from reverting changes to the default network policies.
-Note that {product} won't recreate the default network policies (for example if they're accidentally deleted) if this label is set to `true`.
-Also note that {product} won't create the default network policies if this label is set to `true` when the namespace is created.
-<2> Add this label only if you want to completely remove the default network policies.
-Note that {product} will remove any network policy which matches the name of one of the default policies if this label is set to `true`.
+<1> Adding this label purges all default network policies from the namespace.
+Unless you've already added a network policy this will effectively allow all traffic in and out of the namespace and can be compared to switching off the firewall for the namespace.
+<2> Adding this label removes the default intra-namespace policy, but keeps the base policy that allows system components to connect to the pods.
```

Comment:
I would move this to a separate example, currently this might be confusing to users.